apache_bot_blocker/config.ini

```diff
@@ -0,0 +1,28 @@
+[LOG]
+#Please input your Apache, HTTP, etc. SERVER log file name, path, file format, and timestamp format
+#An IP and Timestamp group MUST be present in the log_file_pattern value, please ensure the value is a REGEX pattern
+log_file_name = digipart-custom.log-20240407.gz
+log_file_path = /home/albert/apache_log
+log_file_format = (?P<ip>\d+\.\d+\.\d+\.\d+) - - \[(?P<timestamp>[^\]]+)\]
+timestamp_format = %%d/%%b/%%Y:%%H:%%M:%%S %%z
+
+
+
+#Add any whitelist domain name by adding the company name as a key and domain hostname as a value
+[GOOD_BOT]
+good_bot = {'Google': ['.googlebot.com', '.google', '.google.com'],
+           'Bing': ['.search.msn.com', '.bing.com', '.search.live.com', '.search.windows.com'],
+           'Yahoo': ['.yahoo.net', '.yahoo.com'], 'DuckDuckGo': ['.duckduckgo.com', '.duckduckbot.com'],
+           'Baidu': ['.baidu.com'], 'facebook': ['.facebook.com', 'fb.com', 'fbcdn.net'],
+           'twitter': ['.twitter.com', '.twimg.com', '.t.co'], 'apple': ['.apple.com'],
+           'amazon': ['.amazon.com']}
+
+
+#Adjust your frequency and interval checks for bot activity detection here
```

Comment:
you must tell the time unit of each value. it is second or ms? Other wise, the configure item name or value must indicate the time unit. for example _interval_1=10s_